cmake/std/atdm/common/toss3/environment_tlcc2.sh

```diff
@@ -25,7 +25,7 @@ module purge
 . /projects/sems/modulefiles/utils/sems-modules-init.sh
 module load sems-env
 module load atdm-env
```

Comment:
Just one suggestion, while we are at it, can we go ahead and remove the load of the 'atdm-env' module and switch to the 'sems-atdm-ninja_fortran/1.8.2' modules like we did for the 'cts1' env?  See:

* https://github.com/trilinos/Trilinos/blob/34f69dc2b81a2e5ae10411bca87c32b2b45b4ec1/cmake/std/atdm/common/toss3/environment_new.sh#L24

That way we get one step closer to removing our dependence on the old 'atdv-env' modules.